metron-platform/metron-pcap/src/main/java/org/apache/metron/pcap/mr/PcapJob.java

```diff
@@ -307,8 +307,11 @@ public void setCompleteCheckInterval(long interval) {
       }
       return this;
     }
-    mrJob.submit();
-    jobStatus.withState(State.SUBMITTED).withDescription("Job submitted").withJobId(mrJob.getJobID().toString());
+    synchronized (this) {
```

Comment:
Can we add a comment about why we need the lock here?

Reply:
Will do. This lock is about thread visibility as opposed to actual issues with concurrent modification. It may be that this lock is not need with getStatus being synchronized. I will double check and report back via modified code and/or code comment on this.

Reply:
fyi, turns out I was right first time around. Synchronization is necessary for visibility in the timer thread that is started after these modifications. I've updated the comments in code to describe this.